content/blog/hello-world.md

```diff
@@ -1,28 +1,40 @@
 
-Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our scope and focus in the very beginning of our journey that we have called {{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}}.
+## What tools are part of scverse?
 
+Our primary experience lies in the field of computational biology with an emphasis on single-cell omics technologies. This experience has defined our initial scope and focus in the very beginning of our journey. Hence, we start with the two most important data structures for single-cell data in the Python ecosystem, [AnnData](https://github.com/scverse/anndata) for uni-modal data and [MuData](https://github.com/scverse/mudata) for multi-modal data, together with [Scanpy](https://github.com/scverse/scanpy) for general purpose single-cell analysis, [muon](https://github.com/scverse/muon) for multimodal single-cell analysis, [scvi-tools](https://github.com/scverse/scvi-tools) for deep probabilistic analysis of single-cell data and [scirpy](https://github.com/scverse/scirpy) for T-cell receptor analysis. We are looking forward to expanding with more major libraries and frameworks which fit our vision soon.
 
-## What this is about
+## What are we doing?
 
-{{< rawhtml >}}<span class="backticked">scverse</span>{{< /rawhtml >}} is a consortium for joint maintenance of core packages in omics data analysis in life sciences. It has been founded to ensure the long-term maintenance of the core infrastructure in the field.
+Our first priority is to build a strong and lasting community around the tools and the associated analysis. The knowledge to build and maintain our tools should be shared among many to avoid single points of failures. Therefore, we are building tooling to more easily generate scverse based packages and analyses to grow the ecosystem. Furthermore, to bring the community together we are planning workshops and hackathons which will be open to the general community. We are also hoping that a healthy scverse community will naturally guide us to develop tooling where it is the most required.
 
+There are already a few visible outcomes of our joint work including improved 
+[mudata performance](https://mudata.readthedocs.io/en/latest/changelog.html#v0-1-2) and
+[mudata documentation](https://mudata.readthedocs.io/en/latest/). Additionally, 
+[scvi-tools is now using MuData](https://github.com/scverse/scvi-tools/pull/1444) for the representation of multimodal data. 
+[Scanpy has gained new workshop notebooks](https://github.com/scverse/scanpy-tutorials/pull/52), and, moreover, we have started 
+[a new page with tutorials for scverse libraries](https://scverse.org/tutorials/).
```

Comment:
Add the new template here

Reply:
@giovp should we mention spatial stuff here as well?